extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/EndpointConfig.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.resteasy.reactive.server.deployment;
+
+import java.util.Objects;
+import java.util.function.BiPredicate;
+
+public class EndpointConfig {
+
+    private String consumes;
+    private String produces;
+    private String endpoint;
+
+    private static final BiPredicate<String, String> isEquivalentMimeType = (item1, item2) -> item1.equals("*")
```

Comment:
This doesn't appear to be used, and it seems simplistic too.

Reply:
Ok. 
So indeed, i have to delete the predicate, and add tests to validate produce mimetype if not explicitely set depending on the method return type...

Reply:
so do we agree that by default if a method doesn't specify it's produce mimetype, it will default to _text/plain_ for primitive type + there object equivalent and String, otherwise it will default to _application/json_ ?

Reply:
If yes, I would like to be able to access https://github.com/quarkusio/quarkus/blob/main/independent-projects/resteasy-reactive/common/processor/src/main/java/org/jboss/resteasy/reactive/common/processor/EndpointIndexer.java#L100 which is a map of all primitive types that will bepicked up as _plain/text_, or maybe there is another map available ?